src/pages/messages/components/TicketDateIntervalFilter.tsx

```diff
@@ -25,22 +25,27 @@ export const TicketDateIntervalFilter = () => {
 
   const onChangeFromDate = (newDate: Date | null) => {
     const newFromDate = newDate ? formatDateStringToISO(newDate) : '';
-    if (!newFromDate && !selectedToDate) {
-      searchParams.delete(TicketSearchParam.CreatedDate);
-    } else {
-      searchParams.set(TicketSearchParam.CreatedDate, `${newFromDate},${selectedToDate}`);
+    // check if year has 4 digits
+    if (newFromDate.match(/^\d{4}-/)) {
+      if (!newFromDate && !selectedToDate) {
+        searchParams.delete(TicketSearchParam.CreatedDate);
```

Comment:
nå er det ikke mulig å resette datosøk her, siden det sjekkes på at strengen må starte med 4 siffer, før det sjekkes for `!newFromDate`